src/Makefile.am

```diff
@@ -156,25 +156,20 @@ eos_updater_SOURCES = \
 
 dist_man8_MANS += docs/eos-updater.8
 
+eos_updater_dbus_if := com.endlessm.Updater
+
 dbusconfdir = $(datadir)/dbus-1/system.d
 dbussystemservicedir = $(datadir)/dbus-1/system-services
 
-dbusconf_DATA = $(eos_updater_dbus_if).conf
-dbussystemservice_DATA = $(eos_updater_dbus_if).service
+dbusconf_DATA = $(srcdir)/$(eos_updater_dbus_if).conf
+dbussystemservice_DATA = $(srcdir)/$(eos_updater_dbus_if).service
 
-CLEANFILES = $(BUILT_SOURCES) $(dbusconf_DATA) $(dbussystemservice_DATA)
+CLEANFILES = $(BUILT_SOURCES)
 
 EXTRA_DIST = \
-	eos-updater-ifname.xslt \
-	eos-updater-policy.xslt \
-	eos-updater-service.xslt \
 	eos-updater.xml \
```

Comment:
It would be useful to also install `eos-updater.xml` as `/usr/share/dbus-1/interfaces/com.endlessm.Updater.xml`. Please do that as a separate commit.